front/app/containers/PasswordChange/index.tsx

```diff
@@ -166,13 +171,13 @@ const ChangePassword = ({ tenant }: Props) => {
                   width="max-content"
                   margin-right="5px"
                   labelMessage={messages.newPasswordLabel}
-                  htmlFor="new_password"
+                  htmlFor="password"
                 />
                 <StyledPasswordIconTooltip />
               </LabelContainer>
-              <PasswordInput name="new_password" autocomplete="new-password" />
-
+              <PasswordInput name="password" autocomplete="new-password" />
```

Comment:
Does autocomplete="new-password" still make sense given the name seems to have been changed to password throughout?